cub/test/internal/catch2_test_integer_utils.cu

```diff
@@ -0,0 +1,140 @@
+// SPDX-FileCopyrightText: Copyright (c) 2025, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+// SPDX-License-Identifier: BSD-3-Clause
+#include <cub/detail/integer_utils.cuh>
+
+#include "c2h/catch2_test_helper.h"
```

Comment:
I am not fully sure, but should this use angle brackets, since it's a different library? (and not a relative include)
```suggestion
#include <c2h/catch2_test_helper.h>
```